src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIInteract.java

```diff
@@ -32,20 +35,60 @@
      * The amount of xp the entity gains per block mined.
      */
     private static final double XP_PER_BLOCK         = 0.05D;
+
     /**
      * The percentage of time needed if we are one level higher.
      */
     private static final double LEVEL_MODIFIER       = 0.85D;
-    /**
-     * The Multiplier to add to hand mining speed.
-     */
-    private static final int    HAND_MINING_MODIFIER = 10;
+
     /**
      * The minimum range the builder has to reach in order to construct or clear.
      */
     private static final int    MIN_WORKING_RANGE    = 12;
 
     /**
+     * Range around the worker to pickup items.
+     */
+    private static final int   ITEM_PICKUP_RANGE       = 3;
+
+    /**
+     * Ticks to wait until discovering that is stuck.
+     */
+    private static final int   STUCK_WAIT_TICKS        = 20;
+
+    /**
+     * The amount of time to wait while walking to items.
+     */
+    private static final int   WAIT_WHILE_WALKING      = 5;
+
+    /**
+     * Horizontal range in which the lumberjack picks up items.
+     */
+    private static final float RANGE_HORIZONTAL_PICKUP = 45.0F;
+
+    /**
+     * Vertical range in which the lumberjack picks up items.
+     */
+    private static final float RANGE_VERTICAL_PICKUP   = 3.0F;
+
+    /**
+     * Number of ticks the lumberjack is standing still.
+     */
+    private              int   stillTicks              = 0;
+
+    /**
+     * Used to store the path index
+     * to check if the lumberjack is still walking.
+     */
+    private              int   previousIndex           = 0;
+
+    /**
+     * Positions of all items that have to be collected.
```

Comment:
If this is the items to be gathered, both the description and the name should mention it. Unless it was an AI solely to to the gathering